Products/ZenModel/migrate/addToolboxLogsToKibana.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import re
+log = logging.getLogger("zen.migrate")
+
+import Migrate
+import servicemigration as sm
+
+sm.require("1.1.5")
+
+class AddToolboxLogsToKibana(Migrate.Step):
+    "Pull logs provided by toolbox utilities to logstash."
+
+    version = Migrate.Version(115, 0, 0)
```

Comment:
There's a new convention for this release regarding the schema versions. See the README in this directory for more information, or grep the other migrations in this directory for `SCHEMA_` to see some examples